src/capability.rs

```diff
@@ -1,46 +1,75 @@
-use crate::{Error, Namespace, Set};
-use std::collections::{BTreeMap, HashMap};
+use crate::{capabilities_to_statement, translation::ToResource, Error};
+use cid::Cid;
+use std::collections::{BTreeMap, BTreeSet};
+
+use crate::ability::{Ability, AbilityName, AbilityNamespace};
 
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
+use serde_with::{serde_as, DisplayFromStr};
 
-fn eq_set_is_empty<T: Eq>(s: &Set<T>) -> bool {
-    s.as_ref().is_empty()
-}
+use iri_string::types::UriString;
+use siwe::Message;
+
+pub type NoteBenes<T> = Vec<BTreeMap<String, T>>;
```

Comment:
I can't find this type in the specs, is it supposed to be Nota Bene? If so, my two years of sleeping in Latin classes tell me you can pluralise n.b.

Reply:
done, it's really only there to stop clippy complaining about overly complex type signatures

Reply:
I still find the name confusing but I trust you! There's still a typo though, should be `NotaBene`

Reply:
fixed the typo 😅